src/getSurfSubModNames.c

```diff
@@ -2082,9 +2082,12 @@ global_model_parameters *getGlobalModelParameters(char *modelVersion, char *TOPO
         GLOBAL_MODEL_PARAMETERS->basin[49] = "Gisborne_v21p11";
         GLOBAL_MODEL_PARAMETERS->basin[50] = "SouthernHawkesBay_v21p12";
         GLOBAL_MODEL_PARAMETERS->basin[51] = "Wairarapa_v21p12";
-        GLOBAL_MODEL_PARAMETERS->basin[52] = "Motu_Bay1_v22p3";
-        GLOBAL_MODEL_PARAMETERS->basin[53] = "Motu_Bay2_v22p3";
-        GLOBAL_MODEL_PARAMETERS->basin[54] = "Motu_Bay3_v22p3";
+        GLOBAL_MODEL_PARAMETERS->basin[52] = "Omaio_Bay1_v22p5";
+        GLOBAL_MODEL_PARAMETERS->basin[53] = "Omaio_Bay2_v22p5";
+        GLOBAL_MODEL_PARAMETERS->basin[54] = "Omaio_Bay3_v22p5";
+        GLOBAL_MODEL_PARAMETERS->basin[55] = "Motu_Bay2_v22p5";
+        GLOBAL_MODEL_PARAMETERS->basin[56] = "Motu_Bay3_v22p5";
+        GLOBAL_MODEL_PARAMETERS->basin[57] = "TolagaBay_v22p5";
```

Comment:
Noting that previously we haven't been consistent either, would it be preferable to keep underscores in the naming consistent from this point onward at least? Omaio and Motu have _ before Bay but Tolaga doesn't. Doesn't matter in the grand scheme of things I guess but could be tidier?